### ISSUE TITLE
fix code examples in hoodie.account API documentation

### ISSUE THREAD

**Issue by gr2m** [hoodiehq/camp#122]
This is a [starter issue](http://hood.ie/blog/starter-issues.html). It’s up for everyone for taking :) New contributors welcome <3

### 🤔 What you will need to know

Our [documentation website](http://docs.hood.ie/en/latest/)’s source code is part of the [hoodie](https://github.com/hoodiehq/hoodie) repository and lives in the [`docs/`](https://github.com/hoodiehq/hoodie/tree/master/docs) folder. Each time there is an update to the repository’s `master` branch, the documentation on [docs.hood.ie](http://docs.hood.ie/en/latest/) gets rebuild automagically through https://readthedocs.org ✨ 

The Documentation is built from `*.rst` files, which stands for [reStructuredText](http://www.sphinx-doc.org/en/stable/rest.html). It’s mostly text and has some special syntax for things like tables or code highlights. Don’t worry if that’s new to you, you’ll get the hang of it 👍 

### 🐞 The Bug

Open the [hoodie.account](http://docs.hood.ie/en/latest/api/client/hoodie.account.html) documentation page. Most examples correctly show `hoodie.account`, but some towards the and only show `account`. For example, `account.profile.get(properties)` should be `hoodie.account.profile.get(properties)`.

### :clipboard: Step by Step
- [x] 🙋 **Claim this issue**: Comment below (or assign yourself and continue at step 4 :)  
  Please 🙏 only claim if you want to start working on it within a day.  
  Once claimed we add you as contributor to this repository.
- [x] 👌 **Accept our invitation** to this repository. Once accepted, assign yourself to this repository
- [x] 👓 **Please review** our [Code of Conduct](http://hood.ie/code-of-conduct/)  
  In a nutshell: be patient and actively kind with us 😊
- [x] 🔄 **replace** the `up for grabs` label with `in progress`.
- [x] You can make the edits directly in the browser if you like. Click the "Edit on GitHub" button on top right of the [`hoodie.account` documentation page](http://docs.hood.ie/en/latest/api/client/hoodie.account.html). Make the changes and commit them with `docs: correct hoodie.account code examples`
- [x] 🔀 **Start a Pull Request**. Mention `closes hoodiehq/camp#122` in the description.  
- [x] 🏁 **Done** :+1: Replace the `in progress` label with `ready`. Ask in comments for a review :)

### 🤔❓ Questions

Ping us in the [Hoodie Chat](http://hood.ie/chat/) or on [Twitter](https://twitter.com/hoodiehq/)

**Comment by lucask42** [comment]
Hello gr2m, I'd like to claim this issue 🙋

**Comment by LowProfileDog** [comment]
Welcome @lucask42 👋🏾🐶 We’ve just invited you to this repository. You can accept the invitation at https://github.com/hoodiehq/camp/invitations and then work off the checklist above. Let us know if you have any questions :)

**Comment by lucask42** [comment]
Great!

**Comment by lucask42** [comment]
Well that was fun!  Please review my pull request :) :) :)

**Comment by gr2m** [comment]
The new version with your fixes is now online at http://docs.hood.ie/en/latest/api/client/hoodie.account.html#hoodie-account-update. Thanks for your help 👍
